components/rhs_thread/rhs_thread.tsx

```diff
@@ -98,7 +100,7 @@ export default class RhsThread extends React.Component<Props, State> {
         window.removeEventListener('resize', this.handleResize);
     }
 
-    public componentDidUpdate(prevProps: Props) {
+    public componentDidUpdate(prevProps: Props, prevState: State, snapshot?: any) {
```

Comment:
`snapshot` is also of type `State` here.

Reply:
True. I was using `any` because I was getting back `any` 😆